html/Callbacks/RT-Extension-TicketActions/Elements/Tabs/Privileged

```diff
@@ -2,7 +2,9 @@
 	my $request_path = $HTML::Mason::Commands::r->path_info;
 	my $actions = PageMenu()->child('actions');
 	my $follow_up_time = RT->Config->Get('TA_FollowUpTime') || '10:00:00';
-	my @follow_up_days = RT->Config->Get('TA_FollowUpDays') || qw(1 3 10);
+	my @follow_up_days = RT->Config->Get('TA_FollowUpDays') || qw(1 7 14);
+	my %skipDow = map { $_ => 1 } @{RT->Config->Get('TA_SkipDow') // [6, 0]};
```

Comment:
Please use the long representation, `SkipDaysOfWork`.

Reply:
Also explain the meaning of `[ 6, 0 ]` ... can we find a better value for this?

Reply:
1. `s/Work/Week/` ?
2. I'll change it to well-known abbreviations.

Reply:
Sorry, `SkipDaysOfWeek`, yes. It would be nice if we could use some string values here, but I doubt that the Perl module allows for that. 0, ..., 6 is not really user friendly.

Reply:
(2) Done.